README.md

```diff
@@ -201,6 +201,13 @@ var railsImagePath = "<%= helpers.image_path('rails.png') %>";
 
 This is enabled by the `rails-erb-loader` loader rule in `config/webpack/shared.js`.
 
+## Ready for Elm
```

Comment:
Yeah, that makes sense.

In terms of being able to scan the document quickly, how about alpha-sorting the options instead? That would improve general organization and easy of scanning the document for available options.